src/App.js

```diff
@@ -5,13 +5,13 @@ import './App.css';
 
 const Home = React.lazy(() => {
   return new Promise(resolve => {
-    setTimeout(() => resolve(import('./components/home')), 4000);
+    setTimeout(() => resolve(import('./components/home')), 2000);
```

Comment:
The timer's were set recently #197 please see this [comment](https://github.com/Namanl2001/MERN-Gurujii-dev/pull/197#discussion_r634404260)

Reply:
I was taking a lot more time to load the components that's why I changed, but nevermind I'll revert that